xml/System.IO.Enumeration/FileSystemEnumerable`1.xml

```diff
@@ -54,10 +54,10 @@
         <Parameter Name="options" Type="System.IO.EnumerationOptions" />
       </Parameters>
       <Docs>
-        <param name="directory">To be added.</param>
-        <param name="transform">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="directory">The path of the directory where the enumeration will be performed.</param>
+        <param name="transform">A delegate method for transforming raw find data into a result.</param>
+        <param name="options">An object describing the enumeration options.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.IO.Enumeration.FileSystemEnumerable`1" /> class with specific search and filtering options.</summary>
         <remarks>To be added.</remarks>
```

Comment:
do you want to add basic exceptions such as ArgumentNullException when directory or transform is null?

Also, it seems that if you pass null to options, its default value will be EnumerationOptions.Default. Worth mentioning?

Reply:
Yes, good idea:

```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

If `options` is passed as `null`, it defaults to <xref:System.IO.EnumerationOptions.Default>.

           ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentNullException"><paramref name="directory" /> or <paramref name="transform" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">The <paramref name="directory" /> path is empty.

-or-

<paramref name="directory" /> contains a null character "\0".</exception>
```